docs/tutorial.md

```diff
@@ -91,31 +90,30 @@ NAME
 echo
 ```
 
-The status of type `Succeeded` shows that the `Task` ran successfully.
+The `Succeeded` status confirms that the `Task` completed with no errors.
+
+To see more detail about the execution of your `Task` view its logs:
 
-To see the actual outcome, use the following command:
 ```bash
 tkn taskrun logs echo-hello-world-task-run
 ```
 
-You will get an output similar to this:
+Below is the example output:
 
 ```
 [echo] hello world
 ```
 
-### Task Inputs and Outputs
+### Specifying the inputs and outputs for your `Task`
 
-In more common scenarios, a `Task` needs multiple steps with input and output
-resources to process. For example a `Task` could fetch source code from a GitHub
-repository and build a Docker image from it.
+In more complex scenarios, a `Task` requires you to define your inputs and outputs. For example a
+`Task` could fetch source code from a GitHub repository and build a Docker image from it.
 
-[`PipelineResources`](resources.md) are used to define the artifacts that can be
-passed in and out of a `Task`. There are a few system defined resource types ready
-to use, and the following are two examples of the resources commonly needed.
+Use one or more [`PipelineResources`](resources.md) to define the artifacts you'd like to pass in
```

Comment:
I prefer the original wording used here.

Reply:
Why? It's a lot harder to parse and is passive. The new wording is clearer and actionable, IMHO. Is it missing a crucial detail? I can re-add the bit about system-defined types if it's absolutely necessary for the reader.

Reply:
I guess I prefer this wording: [`PipelineResources`](resources.md) are used to define the artifacts that can be
passed in and out of a `Task` since the concept is just being introduced for the first time. But I agree that `There are a few system defined resource types ready
to use, and the following are two examples of the resources commonly needed.` can be replaced with `The following are examples of the most commonly needed resources.`

Reply:
In this particular case, the two sentences convey the same exact technical information. The only difference is the passive vs. active voice: "X are used to do Y" vs "use X to do Y". I think the double passive ("X are used to define artifacts to be passed") voice is harder to parse, IMHO.

Reply:
I think it's a more graceful transition into the topic. It's trying to teach the concept, not give an instruction so I think it reads fine as is, but I can leave this alone if it's something you feel strongly one.

I would just ask that `you'd` be replaced with `you would`.

Reply:
Done.